bedrock/stories/templates/stories/includes/macros.html

```diff
@@ -0,0 +1,56 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import card with context %}
+{% from "macros.html" import image with context %}
+
+
+{% macro stories_card(title="", image_url="", link_url="", desc="", class="", image_alt="", include_highres_image=False) %}
+  <section class="c-stories-card {% if class %}{{ class }}{% endif %}">
+    <a class="card-block-link" href="{{ link_url }}">
+      <div class="card-media-wrapper">
+        {{ image(
+          url=image_url,
+          class='card-image',
+          alt=image_alt,
+          loading="lazy",
+          include_highres=include_highres_image,
+          include_l10n=l10n_image
+        ) }}
+      </div>
+      <div class="card-content">
+        <h3 class="card-title"><span>{{ title }}</span></h3>
+        <p class="card-desc">{{ desc }}</p>
+      </div>
+    </a>
+  </section>
+{% endmacro %}
+
+{% macro stories_image(image_url="", alt="", caption="", include_highres=False, image_class="", width="1360", height="765", featured=False, loading=None) %}
+  <figure class="c-stories-image {% if image_class %}{{ image_class }}{% endif %}">
+    {{ image(
+      url=image_url,
+      alt=alt,
+      width=width,
+      height=height,
+      loading=loading,
+      include_highres=include_highres
+    ) }}
+    {% if caption %}
+      <figcaption>{{ caption }}</figcaption>
```

Comment:
This is pretty hard to read on the pink:
![image](https://user-images.githubusercontent.com/205591/181818081-2116e011-ac42-4b07-9195-0619e2469437.png)

If we want a common style that works on different colored backgrounds (but isn't solid black) we could use an rgba color. Eventually we'll likely want a dark theme version of the article header too but that's a problem for the future.